.github/workflows/build-and-publish.yml

```diff
@@ -57,26 +52,24 @@ jobs:
       - name: Download cached dependencies
         uses: Swatinem/rust-cache@v2
         with:
-            shared-key: "linux-cross-builds"
+          shared-key: "linux-cross-builds"
 
       - name: Install cross
         run: cargo install cross
 
       - name: Inject Version
-        working-directory: crates/ev-enclave
+        working-directory: crates/ev-cli
         run: |
           sh ./scripts/insert-cli-version.sh ${{ inputs.full-version }}
 
       - name: Build and Compress cli
         run: |
           mkdir ${{ env.BIN_DIR }}
           mkdir ${{ env.RELEASE_DIR }}
-          cross build -p ev-enclave --release --target ${{ env.LINUX_TARGET }}
-          mv ./target/${{ env.LINUX_TARGET }}/release/ev-enclave ./${{ env.BIN_DIR }}/ev-enclave
-          7z a -ttar -so -an ./${{ env.BIN_DIR }} | 7z a -si ./${{ env.RELEASE_DIR }}/ev-enclave-${{ env.LINUX_TARGET }}-${{ inputs.full-version }}.tar.gz
+          cross build -p ev-cli --release --target ${{ env.LINUX_TARGET }}
+          mv ./target/${{ env.LINUX_TARGET }}/release/ev-cli ./${{ env.BIN_DIR }}/ev
```

Comment:
Probably easier to just add a bin section to cargo toml to output a binary named `ev`?
```
[[bin]]
name = "ev"
path = "src/main.rs"
```

Reply:
nice!